docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: "3.9"
+    
+services:
+  waltz:
+    build: ./
```

Comment:
this should be the image instead, so that it can be used by both the CI and the end-users